src/pages/Books.jsx

```diff
@@ -23,18 +25,20 @@ function Books() {
 
   const [searchVal, setSearchVal] = useState("");
 
-  useEffect(() => {
-    dispatch(fetchBooks());
-  }, [dispatch]);
-
   //Get current posts
   const indexOfLastPost = currentPage * postsPerPage;
-
   const indexOfFirstPost = indexOfLastPost - postsPerPage;
-  const currentPosts = posts.slice(indexOfFirstPost, indexOfLastPost);
-  const howManyPages = Math.ceil(posts.length / postsPerPage);
-
-  const booksArr = currentPosts.map((book) => (
+  const currentPosts = posts?.slice(indexOfFirstPost, indexOfLastPost);
+  const howManyPages = Math.ceil(books.length / postsPerPage);
+  const Genres = [
+    { value: "action", label: `${t("action")}` },
+    { value: "adventure", label: `${t("adventure")}` },
+    { value: "drama", label: `${t("drama")}` },
+    { value: "comedy", label: `${t("comedy")}` },
+    { value: "horror", label: `${t("horror")}` },
+    { value: "romance", label: `${t("romance")}` },
+  ];
```

Comment:
Since this is being used in multiple places at this point, why not make it inside services and important whenever it is needed.

Reply:
I'm not sure about this but I think there is already a genres file in `service`

Reply:
Because we can't add localization to it if it's not inside react component